content/articles/setting-up-a-private-dns-server-with-raspberry-pi/index.md

```diff
@@ -0,0 +1,132 @@
+For computers to speak with one another over the web, all participants within the network must have an IP address: Through these addresses, computers recognize precisely which servers/devices they are communicating with. However, it can be difficult to remember IP addresses and hence the need for DNS servers. Domain Name Systems (DNS) changes domain names into IP addresses. A DNS is like a phone book of web addresses that makes access to pages faster. 
+

```

Comment:
```suggestion
For computers to speak with one another over the web, all participants within the network must have an IP address. Through these addresses, computers recognize precisely which servers/devices they are communicating with. However, it can be difficult to remember IP addresses and hence the need for Domain Name Systems (DNS) servers. DNS servers change domain names into IP addresses. A DNS is like a phone book of web addresses that makes access to pages faster.
```

Reply:
In addition to the above suggestion, you could add an example of a domain name and how it changes to its IP address. Also, add an external link to DNS